### PR TITLE
Profil für Rechnungsworkflow

### PR DESCRIPTION
**Pull request by alexzautke**

### CODE REVIEW

**Comment by SvenSommer** [comment]
Hello @alexzautke,
I have cleaned the repository, eliminating all unnecessary leftover template resources. Furthermore, a ReadMe template has been provided for your convenience. Please use the '${*}' characters to find and replace the specific information for this project.

File: Resources/input/fsh/eabpkv-rechnungsworkflow.fsh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Profile: EABPKVRechnungsworkflow
+Title: "EABPKV Rechnungsworkflow"
+Parent: Task
+Id: eabpkv-rechnungsworkflow
+* extension contains http://hl7.org/fhir/5.0/StructureDefinition/extension-Task.requestedPerformer named requestedPerformer 1..1 MS
```
Comment (florianschoffke) [inline Resources/input/fsh/eabpkv-rechnungsworkflow.fsh:5]: Why this extension? Why did you decide against the usage of .owner?

File: Resources/input/fsh/eabpkv-rechnungsworkflow.fsh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Profile: EABPKVRechnungsworkflow
+Title: "EABPKV Rechnungsworkflow"
+Parent: Task
```
Comment (florianschoffke) [inline Resources/input/fsh/eabpkv-rechnungsworkflow.fsh:3]: Have you considered using the "Claim" Resource instead?

File: Resources/input/fsh/eabpkv-rechnungsworkflow.fsh (hunk 1 above)
Comment (alexzautke) [reply Resources/input/fsh/eabpkv-rechnungsworkflow.fsh:5]: There's an inherent semantical mismatch between the owner element and what the data model in our use case is trying to express. The owner is about the actor who is owning the execution of the task and not who is owning the task itself. See clarification in FHIR R5/R6: https://build.fhir.org/task-definitions.html#Task.owner

File: Resources/input/fsh/eabpkv-rechnungsworkflow.fsh (hunk 2 above)
Comment (alexzautke) [reply Resources/input/fsh/eabpkv-rechnungsworkflow.fsh:3]: This Task profile is meant for tracking the businessStatus of the workflow, for representing the state-machine in a Task profile seemed more appropriate. All information about the services itself will be covered in an Invoice resource. We don't intend to represent the individual services of the Practitioner itself yet in a Claim resource. Only information related to the payment will be available as structured content.

**Review by SvenSommer — APPROVED** [review]
I was able to review this PR from a technical perspective and didn't find any weak points regarding the Gematik FHIR standards. The repository is well set up, GitHub action is working and validating.

However, I'm unable to verify if the content of the profiles is suitable for the intended use case. This needs to be checked by @PeterTabeling-gematik and his team.

**Pull request merged**